src/config/default.js

```diff
@@ -1,7 +1,7 @@
 const config = {
   WEB3_PROVIDER: process.env.REACT_APP_WEB3_PROVIDER || 'https://ropsten.infura.io/v3/608777ea4b3343e291b5ec70d42f2214',
```

Comment:
Value should not be hardcoded

Reply:
This hardcoded value, was already present in the code. Since we've cloned this repo. There is no change from us.

Reply:
This needs to be updated regardless.